From c1778db6b094ba9e6d822257488b59f53718b508 Mon Sep 17 00:00:00 2001
From: Jade Michael Thornton <jademichael@jmthornton.net>
Date: Mon, 17 May 2021 13:36:02 -0500
Subject: [PATCH] fix default tcl command

---
 lisp/progmodes/tcl.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lisp/progmodes/tcl.el b/lisp/progmodes/tcl.el
index f6a50bf1a8..b5696c1664 100644
--- a/lisp/progmodes/tcl.el
+++ b/lisp/progmodes/tcl.el
@@ -169,8 +169,7 @@ taken to mean `smart'.  The default is nil."
   "If not nil, use smart way to find current word, for Tcl help feature."
   :type 'boolean)

-(defcustom tcl-application "wish"
+(defcustom tcl-application "tclsh"
   "Name of Tcl program to run in inferior Tcl mode."
   :type 'string)

--
GitLab
